### PR TITLE
#2561 - Analyzer v2 entry points

### PR DESCRIPTION
**Pull request by jkppr**
ℹ️  _This PR will be merged into the `P17_analyzer_v2_ui` branch! It is part of the [Analyzer v2UI project](https://github.com/google/timesketch/projects/17) and addresses issue #2561 ._

This PR adds the fundamentals for the Analyzers in the new frontend UI.

- It adds the Analyzer Results entry in the Left Panel.
- It adds the entry points for running an analyzer in the left panel and the timeline chip
- It handels empty analyzer result list (e.g. no analyzers run yet)
- It adds some placeholders and TODOs where other components will go that are handled in other issues.

### CODE REVIEW

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResult.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+<template>
+  <div>
+    <!-- TODO: issue #2565 -->
+    <!-- https://github.com/google/timesketch/issues/2565 -->
+  </div>
+</template>
+
+<script>
+// import EventBus from '../../main'
```
Comment (Annoraaq) [inline timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResult.vue:25]: Is this needed?

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  </div>
+</template>
+
+<script>
+// import TsAnalyserResult from './AnalyzerResult.vue'
+
+export default {
+  props: [],
+  components: {
+    // TsAnalyserResult,
```
Comment (Annoraaq) [inline timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue:78]: Is there a reason why this is commented out?

**Review by Annoraaq — APPROVED** [review]
Why is the image uploaded twice? Do we need it in the old UI (timesketch/frontend/dist/no_analyzer_results.png)?

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue (hunk 2 above)
Comment (jkppr) [reply timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue:78]: Good point. I'll fix this.

File: timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResult.vue (hunk 1 above)
Comment (jkppr) [reply timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResult.vue:25]: Not yet, I'll remove it. Thanks

**Comment by jkppr** [comment]
> Why is the image uploaded twice? Do we need it in the old UI (timesketch/frontend/dist/no_analyzer_results.png)?

This is due to a bug in the development setup. The development setup looks for the picture in `127.0.0.1:5000/dist/no_analyzer_results.png` but since the new UI runs on `127.0.0.1:5001` this cannot load the picture.
So as a temporary workaround I uploaded it to both frontends.

@Annoraaq if you have an idea on how to fix that, let me know :)

**Comment by Annoraaq** [comment]
> > Why is the image uploaded twice? Do we need it in the old UI (timesketch/frontend/dist/no_analyzer_results.png)?
> 
> This is due to a bug in the development setup. The development setup looks for the picture in `127.0.0.1:5000/dist/no_analyzer_results.png` but since the new UI runs on `127.0.0.1:5001` this cannot load the picture. So as a temporary workaround I uploaded it to both frontends.
> 
> @Annoraaq if you have an idea on how to fix that, let me know :)

Thanks for explaining 👍

**Pull request merged**